en.json

```diff
@@ -293,12 +300,50 @@
                     }
                 },
                 "native": {
-                    "title": "Desktop Settings"
+                    "title": "Desktop Settings",
+                    "app_behavior": "App Behavior",
+                    "tip_restart": "Some options might require a restart.",
+                    "titlebar": "Titlebar",
+                    "advanced": "Advanced",
+                    "local_dev_mode": "Local Development Mode",
+                    "local_dev_option_title": "I understand there's no going back.",
+                    "local_dev_option_description": "This will change the app to the 'dev' branch, instead loading the app from a local server on your machine.",
+                    "local_dev_option_desc_without_running": "Without a server running,",
+                    "local_dev_option_desc_wont_load": "the app will not load!",
+                    "dev_mode_currently_on": "Development mode is currently on.",
+                    "option": {
+                        "start_with_computer": "Start with computer",
+                        "discord_status": "Enable Discord status",
+                        "nightly": "Revolt Nightly",
+                        "custom_window_frame": "Custom window frame",
+                        "quick_action_btn": "Enable quick action buttons",
+                        "hardware_accel": "Hardware Acceleration"
+                    },
+                    "description": {
+                        "start_with_computer": "Launch Revolt when you log into your computer.",
+                        "discord_status": "Rep Revolt on your Discord status.",
+                        "nightly": "Use the beta branch of Revolt.",
+                        "custom_window_frame": "Let Revolt use its own window frame.",
+                        "quick_action_btn": "Show mute/deafen buttons on the titlebar.",
+                        "hardware_accel": "Use your GPU to render the app, disable if you run into visual issues."
+                    },
+                    "reload_page": "Reload Page",
+                    "reload_app": "Reload App",
+                    "enter_dev_mode": "Enter Development Mode",
+                    "exit_dev_mode": "Exit Development Mode"
                 },
                 "experiments": {
                     "title": "Experiments",
                     "features": "Available Features",
-                    "not_available": "No experiments are available at this time."
+                    "not_available": "No experiments are available at this time.",
+                    "theme_shop": {
+                        "title": "Theme Shop",
+                        "description": "Allows you to access and set user submitted themes."
+                    },
+                    "search": {
+                        "title": "Search",
+                        "description": "Allows you to search for messages in channels."
+                    }
```

Comment:
I think this experiment has been removed and released unflagged? I've never personally seen a search experiment